task_sdk/pyproject.toml

```diff
@@ -27,10 +27,11 @@ dependencies = [
     "httpx>=0.27.0",
     "jinja2>=3.1.4",
     "methodtools>=0.4.7",
-    "msgspec>=0.18.6",
     "psutil>=6.1.0",
     "structlog>=24.4.0",
     "retryhttp>=1.2.0",
+    "orjson>=3.10.11",
+    "msgspec>=0.19.0",
```

Comment:
Does anything in tasksdk import msgspec anymore? I think this might have been the last use of it

Reply:
Yeah there's a few more places. I started to remove those too but then decided not to when I encountered this comment: https://github.com/apache/airflow/blob/f01c53a73573a5dacb2107944d32a5fd731d64f6/task_sdk/src/airflow/sdk/execution_time/supervisor.py#L872

Reply:
I suspect orjson would also be quicker if we can preserve the decoding behaviour.

Reply:
Ah. On that front: I discovered that pendulum v3 (which I don't think we use yet) has it's core written in rust too.

Reply:
This is fine as it is for now then